src/interface/comm.rs

```diff
@@ -198,10 +256,26 @@ impl Socket {
             }
         };
     }
+
+    pub fn setsockopt(&self, level: i32, optname: i32, optval: i32) -> i32 {
```

Comment:
Looking through this, setsockopt doesn't have the same args as lind_setsockopt? How does the optval get changed from a pointer with a length to an i32?

Reply:
This is the libc wrapper, this has nothing to do with what's passed in lind_setsockopt. What you are referring to is the fact that I am not packaging things up in the dispatcher which is the case for both get and set sockopt.  I will fix these now.

Reply:
That has been fixed with the latest PR

Reply:
Yeah, that's what I meant. Cool!